include/ftxui/component/component_options.hpp

```diff
@@ -175,9 +175,12 @@ struct InputOption {
 
   // Style:
   std::function<Element(InputState)> transform;
-  Ref<bool> password = false;  /// < Obscure the input content using '*'.
```

Comment:
I believe the correct fix would be to remove the space in between the "///" and the "<".

See the `Putting documentation after members` section of:
https://www.doxygen.nl/manual/docblocks.html

Could you do this instead?

Reply:
You are right. I made the change and fix the others similar problems.

Reply:
Excellent! Thanks!